resources/locales.clj

```diff
@@ -23,6 +23,7 @@
    "es"
    "ja"
    "cs"
+   "ar"
```

Comment:
was this intentional? Seems weird for something new to show up here

Reply:
oh yeah, not intentional, just a side-effect of me running scripts to test stuff. I'll revert